crates/lang/macro/src/lib.rs

```diff
@@ -515,7 +515,7 @@ pub fn selector_bytes(input: TokenStream) -> TokenStream {
 ///     #[ink(storage)]
 ///     pub struct Erc20 {
 ///         total_supply: Balance,
-///         // more fields ...
+///         // more fields…
```

Comment:
I'd appreciate if there was a space in those cases just what it has been before. or is this bad style that I have to myself adjust personally?